### PR TITLE
#160136206 Create Pull Request Template

### PR DESCRIPTION
**Pull request by Mutuba**
## What does this PR do?
```
Creates a pull request template for group repo, ah-leagueOfLegends.
```

## Description of Task to be completed?
```
Implement a pull request template.

```

## How should this be manually tested?

```
Go to the repo, create a pull request using the template provided.

```
## Any background context you want to provide?

```
Initially, it has been hard to follow PR conventions, but now with the template, it is going to be easier to raise a PR.

```

## What are the relevant pivotal tracker stories?

[ #160136206 ]( link )

## Screenshots if available to support PR
```


```
## Questions:

```


```

### CODE REVIEW

**Review by Derrickkip — APPROVED** [review]
Looks good to me, let's all strive to always use it when creating PR's

**Comment by Teddykavooh** [comment]
https://docs.google.com/document/d/1wV9KNS41w4paebjOfP78MWxbzSbhUpffmTtriRA3rUU/edit
In that link is some other important details that I think should be added.

**Review by Quantum-35 — APPROVED** [review]

**Comment by DerKip** [comment]
Looks nice, ready for use

**Comment by WNjihia** [comment]
Looks good.

**Comment by WNjihia** [comment]
To note:
Ensure your commits adhere to the best practices style guide
